src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -293,6 +293,34 @@
         text: az aks update -g MyResourceGroup -n MyManagedCluster --load-balancer-outbound-ip-prefixes <ip-prefix-resource-id-1,ip-prefix-resource-id-2>
 """
 
+helps['aks kollect'] = """
+    type: command
+    short-summary: Collecting diagnostic information for the Kubernetes cluster.
+    long-summary: |-
+        Collecting diagnostic information for the Kubernetes cluster and store them in the specified storage account.
+        You can provider the storage account in three ways:
+          storage account name and a sas token with write permission.
```

Comment:
I think it will be more clear if we spell out sas as: Shared Access Signature (SAS), here.